frontend/src/global_styles/content/modules/_team_planner.sass

```diff
@@ -62,3 +62,13 @@
   .op-team-planner--wp-loading-skeleton
     svg
       height: 80px
+
+  @media screen and (max-width: 679px)
+    .fc-header-toolbar
+      padding-left: 0
+    .fc-today-button + .fc-button-group
+      display: none
+    .fc-toolbar-chunk:first-child
+      display: none
+    .spot-icon_user + span
```

Comment:
Please avoid styling rules like this if possible. For that specific case you can add the class `hidden-for-mobile` on the span directly and will be hidden.